contracts/README.md

```diff
@@ -50,10 +50,11 @@ $ yarn test
 ```
 
 ## Contributing
+Check out our [Github repo](https://github.com/smartcontractkit/chainlink).
 
-Contributions are welcome! Please refer to
-[Chainlink's contributing guidelines](../docs/CONTRIBUTING.md) for detailed
-contribution information.
+Contributions are welcome! Please refer to the `/docs` folder in our [repo](https://github.com/smartcontractkit/chainlink/tree/develop/docs) where you will find `CONTRIBUTING.md` which has detailed information on how you can contribute.
+
+Also check out our [Developer Wiki](https://github.com/smartcontractkit/chainlink/wiki), especially the "Core Information" and "Core Developer Resources" sections.
```

Comment:
Can we remove this line? I believe the Wiki is out of date and deprecated. Looking at the Wiki homepage, it hasn't been updated since 2020.

Reply:
done.